build.gradle

```diff
@@ -83,3 +83,47 @@ test {
     }
 }
 
+task sourceJar(type: Jar) {
+    from sourceSets.main.allJava
+}
+
+publishing {
+    repositories {
+        maven {
+            credentials {
+                username "${System.env.ARTIUSER}"
```

Comment:
These env. vars are probably only set on `verify-ci` and probably not on `ci-1`, fyi

Reply:
spoke with andy - we should just publish to local maven repository and magic does the rest